services/frontend/src/components/profileForms/primaryInfoForm/PrimaryInfoFormView.jsx

```diff
@@ -488,7 +488,23 @@ const PrimaryInfoFormView = ({
             <Col className="gutter-row" xs={24} md={8} lg={8} xl={8}>
               <Form.Item
                 name="pri"
-                label={<FormattedMessage id="pri" />}
+                label={
+                  <>
+                    <FormattedMessage id="pri" />
+                    <div className="prim-popoverStyleCareer">
+                      <Popover
+                        trigger={["focus", "hover"]}
+                        content={
+                          <div className="prim-popoverStyle">
+                            <FormattedMessage id="pri.private" />
+                          </div>
+                        }
+                      >
+                        <InfoCircleOutlined tabIndex={0} />
+                      </Popover>
```

Comment:
The screen reader does not actually pick up the popovers. I'm not sure what the fix and needs to fixed with all the popovers so it could be done in a separate PR

Reply:
@Rizvida1 thoughts?

Reply:
Those don't seem accessible either. They are like the popover.